src/git-hack

```diff
@@ -26,7 +30,11 @@ function steps {
 
   echo "checkout_main_branch"
   echo "rebase_tracking_branch"
-  echo "push"
+
```

Comment:
True but I think this is nicer to read since the if-block takes up 3 lines. Perhaps we can add an `echo_if_true($1=string, $2=conditional)` for concision.